toolbox/fdc3-conformance/Intents-Tests.md

```diff
@@ -0,0 +1,240 @@
+  - Then in the first step, use `fdc3.raiseIntent(‘aTestingIntent’, testContextX, appIdentifier)` to target the running instance of app A, otherwise, as above.
+- `2.0-RaiseIntentTargetedInstanceResolveFindInstances`: Repeat the above test, but:
+  - Before the first step, use `let appIdentifier = await fdc3.open({appId: "<A's appId>"})` to start A and retrieve its `AppIdentifier` with instance details.
+  - Then in the first step, use `fdc3.raiseIntent(‘aTestingIntent’, testContextX, appIdentifier)` to target the running instance of app A, otherwise, as above.
+- `2.0-RaiseIntentFailedResolve`: Perform above test, but:
+  - Use `fdc3.raiseIntent(‘aTestingIntent’, testContextY)`.  Note that no app supports this intent and context combination.**
+  - You should receive a JavaScript Error with the message `ResolveError.NoAppsFound`.
+- `2.0-RaiseIntentFailTargetedAppResolve1`: Perform above test, but:** 
+  - Use `fdc3.raiseIntent(‘aTestingIntent’, testContextY, {appId: "<A's appId>"})`.
+  - You should receive a JavaScript Error with the message `ResolveError.NoAppsFound`.
+- `2.0-RaiseIntentFailTargetedAppResolve2`: Perform above test, but:
+  - Use `fdc3.raiseIntent(‘aTestingIntent’, testContextY, {appId: "NonExistentApp"})`.
+  - You should receive a JavaScript Error with the message `ResolveError.TargetAppUnavailable `.
+- `2.0-RaiseIntentFailTargetedAppResolve3`: Perform above test, but: 
+  - Use `fdc3.raiseIntent(‘sharedTestingIntent2’, testContextY, {appId: "<H's appId>"})`.
+  - You should receive a JavaScript Error with the message `ResolveError.IntentDeliveryFailed` (as this app is configured for the intent and context pair, but does not add any intent listeners). 
+  - **Note:  Test will need an extended timeout to allow for this to be returned in time by the desktop agent, which will have a vendor-defined timeout.**
+- `2.0-RaiseIntentFailTargetedAppResolve4`: Perform above test, but: 
+  - `fdc3.raiseIntent(‘sharedTestingIntent2’, testContextY, {appId: "<I's appId>"})`
+  - You should receive a JavaScript Error with the message `ResolveError.IntentDeliveryFailed` (as this app is configured for the intent and context pair, but adds intent listeners of the wrong type.
+  - **Note:  Test will need an extended timeout to allow for this to be returned in time by the desktop agent, which will have a vendor-defined timeout.**
+- `2.0-RaiseIntentFailTargetedAppInstanceResolve1`: Perform above test, but:
+  - First spawn an instance of App **A** and collect its `AppIdentifier` with `const appIdentifier = await fdc3.open({appId: "<A's appId>"})`.
+  - Then use `fdc3.raiseIntent(‘aTestingIntent’, testContextY, appIdentifier )` to target that instance.  
+  - You should receive a JavaScript Error with the message `ResolveError.NoAppsFound` (since A doesn't support this context type).
+- `2.0-RaiseIntentFailTargetedAppInstanceResolve2`: Perform above test, but:
```

Comment:
in `2.0-RaiseIntentFailTargetedAppInstanceResolve2`: change `testContextY` to `testContextX`